index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <img id="shape1" src="/assets/Shape.png">
+        <img id="shape2" src="/assets/Shape2.png">
+        <img id="menupng" src="/assets/ic_menu.png">
```

Comment:
- [ ] The menu button will be developed later and should be wrapped in a `nav` tag for semantics reasons. Kindly wrap the image you used for the mobile menu button.
 - [ ] The 2 shape images you used should be background-images and should not be part of the content of the page. If an image is not part of the content use CSS background-image. Kindly remove them and use the `background-image` attribute in CSS and [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
Hii 👋 @leonardodiasb 
Thanks for the suggestions I will like to implement those changes.